plugins/slick.state.js

```diff
@@ -126,12 +131,44 @@
 
                 _grid.setColumns(state.columns);
               }
+              setUserData(state.userData, true);
             }
             dfd.resolve(state);
           }, dfd.reject);
       });
     }
 
+    function setUserData(data, comesFromState)
```

Comment:
maybe you should add JSDocs comment on top of each function because I'm not exactly sure what each function/arguments does. Also I would prefer to see the curly brace at the end of each `if` not below it which is the default JS standard

Reply:
I need a bit of time to document the functions, but will surely do. will put WIP in the name again...

Reply:
@ghiscoding could you please check if the documentation corresponds to your expectations? :)